recipes/mafft/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "mafft" %}
```

Comment:
Can you add a comment somewhere saying that the recipe comes bioconda?

Reply:
Done